Makefile

```diff
@@ -127,11 +127,11 @@ new_env: clean
 		echo "In a virtual environment! Exit first: 'exit'.";\
 	fi
 protolint_install:
-	GO111MODULE=on GOPATH=~/go go get -u -v github.com/yoheimuta/protolint/cmd/protolint@v0.27.0
+	GO111MODULE=on GOPATH=~/go go install -v github.com/yoheimuta/protolint/cmd/protolint@v0.27.0
```

Comment:
I mean if this works with 1.17.7 its all fine with me. Does it allow for use of either get or install?

Reply:
works